domain/robstoll-lib/atrium-domain-robstoll-lib-common/src/main/kotlin/ch/tutteli/atrium/domain/robstoll/lib/creating/iterableAssertions.kt

```diff
@@ -70,9 +68,9 @@ fun <E : Any, T : Iterable<E?>> _iterableAll(
     }
 }
 
-fun _hasNext(subjectProvider: SubjectProvider<Iterable<*>>): Assertion =
-    ExpectImpl.builder.createDescriptive(subjectProvider,
-        DescriptionBasic.HAS, RawString.create(DescriptionIterableAssertion.NEXT)) { it.iterator().hasNext() }
+fun <E : Any> _hasNext(expect: Expect<Iterable<E>>): Assertion =
+    ExpectImpl.builder.createDescriptive(expect,
+        DescriptionBasic.HAS_ELEMENT, null) { it.iterator().hasNext() }
```

Comment:
this look as follows in reporting:
``` 
* has a next element: 
```
The way you did it before was better (and correct), it resulted in
```
* has: a next element
```